### DISCUSSION TITLE
Is haversine_bearing supposed to return -180 to 180?

### DISCUSSION THREAD

**Discussion by kingkong-cmd** [georust/geo#1225] [Q&A] [answered]
I was expecting to get values from 0 to 360. Is -180 to 180 a convention?

**Comment by michaelkirk** [comment, marked as answer]
Timely! I just merged https://github.com/georust/geo/pull/1216 2 days ago.

It hasn't been released yet, but if you are using `main` you can use the new `Haversine::bearing(a, b)` to get output in `0...360`

See #1210 for more.

**Reply by kingkong-cmd** [reply]
Aha, sorry, didnt check the commits. Thats great, and i will just wait for next release. I believe a simple `%% 360` fixes it in the meantime.

**Reply by michaelkirk** [reply]
I think that should be: `(bearing + 360.0) % 360.0`

**Reply by grevgeny** [reply]
It’s like a remainder but for Euclidean division, so `bearing.rem_euclid(360.0)` will work as well to get an output in `0...360`

**Reply by kingkong-cmd** [reply]
Apologies, I thought rust had a modulo operator %%, but I see now that is not the case. Then I will use one of your suggestions instead.

Edit: I also read https://github.com/georust/geo/issues/1210 and I wholeheartedly support using stupid and explicit language for docs. Makes life so much easier for idiots like me.
